makefile

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/make
```

Comment:
Happy to drop this, I simply noted it was in Smoothie1 and not Smoothie2 so ported it.